arm/.global/global.module.tests.ps1

```diff
@@ -150,9 +156,17 @@ Describe 'Readme tests' -Tag Readme {
                 if (Test-Path (Join-Path $moduleFolderPath 'deploy.bicep')) {
                     $templateFilePath = Join-Path $moduleFolderPath 'deploy.bicep'
                     $templateContent = az bicep build --file $templateFilePath --stdout | ConvertFrom-Json -AsHashtable
+
+                    if (-not $templateContent) {
+                        throw "Unable to compile the deploy.bicep template's content. This can happen if there is an error in the template. Please check if you can run the command `$templateContent = az bicep build --file $templateFilePath --stdout | ConvertFrom-Json -AsHashtable`."
```

Comment:
This is thrown by multiple tests. Should we create a custom exception to be reused?

Reply:
Hey @eriqua, do you mean like a variable outside of all describe blocks that we then pass in via `$script:compilationException = '(...)'?